src/site/utilities/featureFlags.js

```diff
@@ -35,6 +39,7 @@ const flagsByBuildtype = {
     featureFlags.FEATURE_REGION_DETAIL_PAGE_FEATURED_CONTENT,
   ],
   vagovstaging: [
+    featureFlags.FEATURE_FIELD_ADDITIONAL_INFO,
```

Comment:
I think we want to remove this flag from staging and prod, since the change isn't in those environments yet.